### PR TITLE
[NP-10689] Bundle login and controller support in a single method

### PR DESCRIPTION
**Pull request by lchanmann**

### CODE REVIEW

File: src/foam/nanos/controller/ApplicationController.js [hunk 1]
```diff
@@ -813,6 +814,12 @@ foam.CLASS({
       });
     },
 
+    async function login(identifier, password) {
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/ApplicationController.js:817]: Given that we are trying to move stuff out of ApplicationController, why is this needed?

Comment (lchanmann) [reply src/foam/nanos/controller/ApplicationController.js:817]: Some methods on the controller are needed to be called right after login but they are not exported and currently we were just importing `ctrl` then call them manually. Bundle them in a single method so that it doesn't need to call these un-exposed methods.
